Doc/c-api/exceptions.rst

```diff
@@ -461,10 +462,10 @@ Querying the error indicator
 
    .. deprecated:: 3.12
 
+      Use :c:func:`PyErr_GetRaisedException` instead.
```

Comment:
I suggest moving this directive to the top of the `.. c:function` content block, like the original message was, so users have a heads-up right away that the function is deprecated and don't have to through the whole description only to find a notice that it is deprecated and they should use something else buried at the bottom.